components/SpotCard.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import Image from "next/image";
+import placeholderSpotImage from "../assets/images/card-image-placeholder.jpg";
+
+export default function SpotCard(spot) {
+  return (
+    <Card>
+      <SpotImage src={placeholderSpotImage} alt="Placeholder Spot Image" />
+      <SpotInfos>
+        <SpotCategory>{spot.spot.category}</SpotCategory>
+        <SpotName>"{spot.spot.name}"</SpotName>
+      </SpotInfos>
+    </Card>
+  );
+}
```

Comment:
This is totally up to you, but I just wanted to remind you, that you can also destructure `spot` directly from the props object. It makes the code a bit more tidy 🧹 
```suggestion
export default function SpotCard({spot}) {
  return (
    <Card>
      <SpotImage src={placeholderSpotImage} alt="Placeholder Spot Image" />
      <SpotInfos>
        <SpotCategory>{spot.category}</SpotCategory>
        <SpotName>"{spot.name}"</SpotName>
      </SpotInfos>
    </Card>
  );
}
```